tools/build/build.js

```diff
@@ -188,6 +188,7 @@ export const DmTarget = new Juke.Target({
   ],
   inputs: [
     '_maps/map_files/generic/**',
+    'maps/**/*.dm'
```

Comment:
need's a comma `,` at the end as well fyi

Reply:
```suggestion
    'maps/**/*.dm',
```